synths/core-modules.scd

```diff
@@ -35,15 +35,19 @@ this may be refacored later.
 
 ~dirt.addModule('sound',
 	{ |dirtEvent|
-		var instrument, args;
+		var instrument, freq, args;
 		if(~diversion.value.isNil) {
 			if(~buffer.notNil) {
+				freq = ~freq;
+				if(~tune.notNil and: { ~tune != 0 }) {
+					freq = freq * (~metaData !? _.baseFreqToFreqRatio ? 1.0);
```

Comment:
since `metaData` is a Dictionary, better write:
```
freq = freq * (~metaData !? _[\baseFreqToFreqRatio] ? 1.0);
```
Or, in a way, it is clearer and more efficient:
```
if(~tune.notNil and: { ~tune != 0 } and: { ~metaData.notNil }) {
freq = freq * (metaData[\baseFreqToFreqRatio] ? 1.0) }
}
```